CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ the detailed section referring to by linking pull requests or issues.
 ### Detailed Changes
 
 #### Added
+- Additional Http Datasink Properties to the Contract Agreement Transfer Dialog Form
```

Comment:
missing verb, i.e. "Added additional ...", also you can shorten "Contract Agreement Transfer Dialog" to just "Transfer Dialog"